src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -11,6 +19,17 @@
  * A mapper class for mapping examinee data to the JAXB {@link tds.exam.results.trt.TDSReport.Examinee} object
  */
 public class ExamineeMapper {
+    private static final String DEFAULT_BIRTH_DATE = "1900-01-01";
+    private static final String DOB_ATTRIBUTE_ID = "DOB";
+    private static final Logger log = LoggerFactory.getLogger(ExamineeMapper.class);
+
+    private static final ImmutableMap<String, String> attributeNames = ImmutableMap.of(
```

Comment:
minor nit: the variable only needs to be Map<String, String>  

In general, i like to use the least specific container possible when declaring variables.